src/test/java/com/salesforce/dynamodbv2/UpdateTest.java

```diff
@@ -134,10 +228,38 @@ void updateSetFieldMultipleTimesFails(TestArgument testArgument) {
         });
     }
 
+    @ParameterizedTest
+    @ArgumentsSource(DefaultArgumentProvider.class)
+    @DefaultArgumentProviderConfig(tables = { TABLE1 })
+    void updateAddFieldMultipleTimesFails(TestArgument testArgument) {
+        testArgument.forEachOrgContext(org -> {
+            Map<String, AttributeValue> updateItemKey = ItemBuilder.builder(testArgument.getHashKeyAttrType(),
+                HASH_KEY_VALUE)
+                .build();
+
+            UpdateItemRequest updateItemRequest = new UpdateItemRequest()
+                .withTableName(TABLE1)
+                .withKey(updateItemKey)
+                .withUpdateExpression("add someField :someValue, " + SOME_FIELD + " :someValue")
```

Comment:
Can you use `SOME_FIELD` both times? I had to lookup `SOME_FIELD` definition to confirm it is actually `someField`.